internal/pkg/envconfig/envconfig.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// TODO: there proboly already is a package for this... Where?
+
+package envconfig
+
+import (
+	"fmt"
+	"os"
+)
+
+func NoCache() bool {
+	env := os.Getenv("SINGULARITY_NO_CACHE")
+
+	fmt.Println("DEBIG: ENV: ", env)
+
+	if env == "1" || env == "true" {
```

Comment:
Should `SINGULARITY_NO_CACHE` work if it's set to _anything_ ... i.e. as long as `SINGULARITY_NO_CACHE` is set, then don't cache anything.

Reply:
Okay, I'll fix that :+1: Thanks!

Reply:
Although, wont it be confusing to new users if they set `SINGULARIY_DISABLE_CACHE=false`, they might expect it to be off, (disabled) ?...

Reply:
If the default is to cache ... why would you set `SINGULARITY_DISABLE_CACHE=false` ? Doesn't that seem ... counter intuitive to define something that it you do by default?

Reply:
Yeah, agree, I was just thinking out loud.

Reply:
Agree. It may be wanted to have a specific value like the `1` or `true` ... From my point of view though in some cases like this it's enough that the env variable is set ... not that it's set to a specific value. Hell, it could be set to `Bob` ... and IMO we should still not cache, because the variable is there.